solutions/019.md

```diff
@@ -78,3 +78,20 @@ class Solution:
 ```
 
 ***NB***: If you want to get community points please suggest solutions in other languages as merge requests.
+
```

Comment:
Please improve the following things: 
- Please add an explanation to your implementation using the following format:
 ### <Solution name>
<solution explanation>
#### Java
```java
<code_here>
``` 
***NB***: If you want to get community points please suggest solutions in other languages as merge requests.